wandb/integration/gym/__init__.py

```diff
@@ -32,12 +32,13 @@ def monitor():
 
     def close(self):
         recorder.orig_close(self)
-        m = re.match(r".+(video\.\d+).+", getattr(self, path))
-        if m:
-            key = m.group(1)
-        else:
-            key = "videos"
-        wandb.log({key: wandb.Video(getattr(self, path))})
+        if wandb.run:
+            m = re.match(r".+(video\.\d+).+", getattr(self, path))
+            if m:
+                key = m.group(1)
+            else:
+                key = "videos"
```

Comment:
nit: just to make it more pythonic
```suggestion
            key = m.group(1) if m else "videos"
```